website/static/js/fangorn.js

```diff
@@ -141,6 +141,12 @@ function cancelUpload(row) {
     tb.isUploading(handled && filesArr.length > 1);
 }
 
+function removeFromUI(file, tb) {
```

Comment:
Super tiny nit: The ordering of arguments seems backwards. As a rule of thumb, the first argument is the main thing being acted upon. In this case, that's the treebeard. The arguments that follow are the parameters to the action (in this case, the file).

Definitely not a blocker, though.